ktpinblock/ktpinblock/src/commonMain/kotlin/org/superarts/ktpinblock/PinBlock.kt

```diff
@@ -7,12 +7,14 @@ import org.superarts.ktpinblock.format.PinBlockFormat
 import org.superarts.ktpinblock.format.iso.EftInputValidator
 import org.superarts.ktpinblock.utility.toHexString
 import kotlin.experimental.or
+import kotlin.js.JsExport
 
 /**
  * Implements PinBlockEncoder and PinBlockDecoder based on different PinBlockFormats.
  * Internal calculation is based on bytes, i.e. `encodeToBytes`.
  * To combine hi and low nibbles together, use `encodeToCompactBytes`.
  */
+@JsExport
```

Comment:
You don't have to `JsExport` every APIs. Most `internal` APIs are not needed. However, if you have some problems accessing some JS codes, try to `JsExport` more components.